tests/OSPSuite.SimModel.Tests/OSPSuite.SimModel.Tests.vcxproj

```diff
@@ -149,14 +149,14 @@
     <Link>
       <AdditionalDependencies>DCI6_1.lib;OSPSuite.FuncParser.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <OutputFile>$(OutDir)$(TargetName).dll</OutputFile>
-      <AdditionalLibraryDirectories>$(SolutionDir)pre\DCI\$(Configuration);$(SolverPath)bin\native\x86\$(Configuration);$(FuncParserPath)bin\native\x86\$(Configuration);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>$(ProjectDir)..\..\src\DCI\DCI_6.1\ReleaseX64;$(SolverPath)bin\native\x64\$(Configuration);$(FuncParserPath)bin\native\x64\$(Configuration);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <AssemblyDebug>true</AssemblyDebug>
       <SubSystem>Windows</SubSystem>
       <TargetMachine>MachineX64</TargetMachine>
     </Link>
     <PostBuildEvent>
-      <Command>copy "$(ProjectDir)..\..\src\DCI\$(Platform)\$(Configuration)\*.dll" "$(OutputPath)"</Command>
+      <Command>copy "$(ProjectDir)..\..\src\DCI\DCI_6.1\ReleaseX64\*.dll" "$(OutputPath)"</Command>
```

Comment:
Why is DCI not following the release convention (e.g. Release/x86 and Release/x64 or sthg like that?)